controllers/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -675,57 +877,44 @@ func TestModifyUserAccounts(t *testing.T) {
 func TestSyncMurStatusWithUserAccountStatuses(t *testing.T) {
 	logf.SetLogger(zap.New(zap.UseDevMode(true)))
 	s := apiScheme(t)
+	spaceBinding := spacebindingtest.NewSpaceBinding("john", "john-space", "admin", "john-123")
+	space := spacetest.NewSpace(commontest.HostOperatorNs, "john-space",
+		spacetest.WithLabel(toolchainv1alpha1.SpaceCreatorLabelKey, "john-123"),
+		spacetest.WithSpecTargetCluster(commontest.MemberClusterName))
 
 	t.Run("mur status synced with updated user account statuses", func(t *testing.T) {
 		// given
 		// setup MUR that wil contain UserAccountStatusEmbedded fields for UserAccounts from commontest.Member2ClusterName and "member3-cluster" but will miss from commontest.MemberClusterName
 		// then the reconcile should add the misssing UserAccountStatusEmbedded for the missing commontest.MemberClusterName cluster without updating anything else
```

Comment:
it seems that the comment is outdated now, since there is not member3-cluster involved anymore.